app/src/main/java/com/example/poste/http/MyApiService.java

```diff
@@ -12,6 +13,12 @@ public interface MyApiService {
     @POST("users/")
     Call<ResponseBody> registerUser(@Body RegisterRequest registerRequest);
 
+    @POST("posts/")
+    Call<ResponseBody> createPost(@Body CreatePost createPost);
```

Comment:
this should also pass the authorization header the way getData does; this is how we tell the backend what user we're talking about (and ensures the user is authorized to perform the action)

In other words, this should pass both a @Body CreatePost, and a @Header("Authorization") String authToken. You can see in the getData method how that authToken is populated

Reply:
I'm gonna need some one on one time with you tomorrow on this.
I don't think I understand what to do here. reach out on slack and we can set a time.

Reply:
Feel free to DM me. There's also a new PR up that shows a post deletion call. Happy to discuss or make a video.